src/aura/STG_CMP_Tabs/STG_CMP_TabsHelper.js

```diff
@@ -239,6 +242,41 @@
         });
         $A.enqueueAction(action);
     },
+    
+    getpreferredPhonePicklistEntries : function(component, preferredValue) {
+        //Get all active Contact Phone data type picklist entries
+        var action = component.get("c.getPhoneFields");
+        var prefix = component.get("v.namespacePrefix");
+        var objectName = prefix +'Contact';
```

Comment:
There will never be a namespace on the Contact object since it's a standard Salesforce field. This will probably fail in a namespaced org.

Reply:
Confirmed in a dev namespaced scratch org - no values appear in the picklist.